app/src/main/java/org/kiwix/kiwixmobile/downloader/DownloadService.java

```diff
@@ -230,6 +236,7 @@ public void pauseDownload(int notificationID) {
   }
 
   public boolean playDownload(int notificationID) {
+    Log.i("kiwixdownloadservice", "Starting ZIM Download");
```

Comment:
Ditto, see line 183, be good to know more about what's being downloaded if practical.